app/templates/session-expired.html

```diff
@@ -5,12 +5,16 @@
 
 {% block main %}
 
-  <div class="u-mb-s">
+<div class="u-mb-s">
     <h1 class="saturn" {{helpers.track('visible', 'Timeout', 'Timeout expired')}}>Your session has expired</h1>
-  </div>
+</div>
 
   <div class="panel panel--simple panel--success panel--spacious">
     <p>To help protect your information we have signed you out.</p>
   </div>
 
+<br>
```

Comment:
Use a margin utility class (eg `u-mb-s`) on the `panel` instead of a `<br>`. Only use `<br>` for line-breaks in text, not to visually separate content.